docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3.7'
+
+services:
+  opencollective_api:
+    build:
+      context: . # Build the api image from the Dockerfile
+    command: ${OPENCOLLECTIVE_DOCKER_INIT_CMD} # Use this env var to send pre-defined and custom commands
```

Comment:
Currently running `docker-compose up` generates a warning:

> WARNING: The OPENCOLLECTIVE_DOCKER_INIT_CMD variable is not set. Defaulting to a blank string.

Maybe we should add a default value here to avoid that?

Reply:
Sure. Working on it.

Reply:
Great catch. Working on it.

Reply:
Done!